### ISSUE TITLE
underglow LED support

### ISSUE THREAD

**Issue by mattvenn** [keyboardio/Chrysalis#379]
Dygma Raise has underglow LEDs unrelated to the keys. 
There are 30 on the left side and 32 on the right.
They are updated in the same way as normal LEDs but have different indexes.

Underglow LEDs should have the same support for key LEDs.

![image](https://user-images.githubusercontent.com/634220/57145175-ec79d080-6dc2-11e9-83ac-6f9835e005f1.png)

**Comment by TreTuna** [comment]
Hmm... This is interesting because the layer makes the keyboard panel huge, especially when you tack on the editor panel at the bottom of that. Shouldn't be too bad, though I know we've had some issues with the window height before.

**Comment by mattvenn** [comment]
Yes, seen your other comments on the size of the window. Won't post more about it here ;)

**Comment by Simon-Claudius** [comment]
I'll take care of this probably sometime this week.

**Comment by obra** [comment]
No supported keyboards have underglow
